### ISSUE TITLE
black bar on top of android app when viewing images

### ISSUE THREAD

**Issue by Raptor-TK** [BlueBubblesApp/bluebubbles-app#2869]
BlueBubbles Version Number: 1.15.0 Version Code: 2070

There is a black bar on top of android app when viewing pictures.

Especially noticable when zoomed in:
![Screenshot_20241206_152650_BlueBubbles](https://github.com/user-attachments/assets/eb24f34a-9005-49f2-9093-dbb1e366a0cd)

**Comment by SpaceSaver** [comment]
Are you using the Material theme?

**Comment by Raptor-TK** [comment]
Yes, using material theme, but even switching to the iOS and Samsung themes also give the same problem of the black bar on top.

**Comment by zlshames** [comment]
I _think_ i may have fixed this in the app rewrite (beta). If you are able to try the beta, then that would confirm it. @Raptor-TK

**Comment by Raptor-TK** [comment]
Tested on latest beta and unfortunately it is _not_ fixed.

<img width="1080" height="2520" alt="Image" src="https://github.com/user-attachments/assets/d8256627-03e4-418f-a6d2-56c17685534f" />

**Comment by zlshames** [comment]
> Tested on latest beta and unfortunately it is _not_ fixed.
> 
> <img alt="Image" width="857" height="2000" src="https://private-user-images.githubusercontent.com/53501197/581535180-d8256627-03e4-418f-a6d2-56c17685534f.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzY3OTE4OTQsIm5iZiI6MTc3Njc5MTU5NCwicGF0aCI6Ii81MzUwMTE5Ny81ODE1MzUxODAtZDgyNTY2MjctMDNlNC00MThmLWE2ZDItNTZjMTc2ODU1MzRmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDIxVDE3MTMxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY0YTQ0ZjljZGI1MzVlZDA5NGY0NDJkMzUzMGUwN2Q3OWIxZWI1NTJhNjk4MWI2MGJmZjljNjgyZWUwNjFiNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRmpwZWcifQ.P8x-yNWL2_YHJBXGJGbGb2aLhsXQSLYV72N1_KQ-RIY">

Seems to me like this is just due to your device dimensions compared to the image. What do you _want_ it to look like? Zoom into the image? Stretch the image? We have to be able to maintain aspect ratio and show the entire image.

**Comment by zlshames** [comment]
I see in the original screenshot that the controls are anchored to the top of the screenshot. So maybe it's just a matter of moving it to the top rather than the top of the image

**Comment by Raptor-TK** [comment]
I tested on multiple android devices. Also happens on a sony xperia 1vii as well as a samsung fold 7 and samsung fold 6.

I zoomed in and tried moving to the top but the top is always a black bar and i can't see the full picture as the black bar is always filling up the top.

**Comment by Raptor-TK** [comment]
The way to reproduce is to open a picture, tap once to hide the ui elements, double tap to zoom in, pinch to zoom right in. Move the zoomed in picture around to try fill the screen.

Then you notice the top there is aways a section (black bar) where the image will not display on as if if was reserved only for some gui elements and the image can not be displayed in that spot.

**Comment by Raptor-TK** [comment]
The latest beta version (newer one than the one when I lasted tested) now has fixed this issue, so I'm closing the bug report as resolved.
